### PR TITLE
Added support for LocationIQ

### PR DESCRIPTION
**Pull request by theju**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 <div style="text-align:center"><img src="doc/install.jpg" alt="Installing Plugin"></div>
 
 
-## Bulk Nominatim Tools
+## LocationIQ Tools
 
-The plugin ***Settings*** need to be configured to point to a nominatim endpoint. See the ***Settings*** section on how to do this. The plugin can be accessed from the `Plugins->Nominatim GeoCoding` menu or from the two tool bar icons.
+The plugin ***Settings*** need to be configured to point to a nominatim endpoint. See the ***Settings*** section on how to do this. The plugin can be accessed from the `Plugins->LocationIQ GeoCoding` menu or from the two tool bar icons.
 
 <div style="text-align:center"><img src="doc/menu.jpg" alt="Nominatim GeoCoding"></div>
```
Comment (gopi-ar) [inline readme.md:15]: This image needs to be updated

File: readme.md [hunk 2]
```diff
@@ -49,8 +49,4 @@ Clicking on the ***OK*** button causes the plugin to start geocoding.
...
 
 ## Settings
-In ***Settings*** the user can select the Nominatim Service URL endpoint, the maximum number of addresses to geocode and for reverse geocoding the level of detail where 0 represents the country and 18 the address number. Here is the dialog window.
-
-<div style="text-align:center"><img src="doc/settings.jpg" alt="Settings"></div>
-
-Please note that this plugin is designed to use commercial or personal Nominatim services. The OpenStreetMap URL displayed here is not for bulk use as it violates their policy and will result in the user being blocked from their site for a period of time.
\ No newline at end of file
+In ***Settings*** the user can configure the custom parameters for geocoding/reverse geocoding. The documentation for these parameters are found at https://locationiq.com/docs?ref=qgis
```
Comment (gopi-ar) [inline readme.md:52]: This can be a proper link w/ different text?

File: bulkNominatim.py [hunk 3]
```diff
@@ -68,18 +68,18 @@ def unsetTool(self, tool):
     def unload(self):
         """Unload BulkNominatim from the QGIS interface."""
         self.canvas.unsetMapTool(self.reverseGeocodeTool)
-        self.iface.removePluginMenu(u"Nominatim GeoCoding", self.nominatimAction)
-        self.iface.removePluginMenu(u"Nominatim GeoCoding", self.reverseGeocodeAction)
-        self.iface.removePluginMenu(u"Nominatim GeoCoding", self.settingsAction)
-        self.iface.removePluginMenu(u"Nominatim GeoCoding", self.helpAction)
+        self.iface.removePluginMenu(u"LocationIQ GeoCoding", self.nominatimAction)
+        self.iface.removePluginMenu(u"LocationIQ GeoCoding", self.reverseGeocodeAction)
```
Comment (gopi-ar) [inline bulkNominatim.py:72]: How do we toggle this off?

File: bulkNominatim.py [hunk 4]
```diff
@@ -68,18 +68,18 @@ def unsetTool(self, tool):
...
+        self.iface.removePluginMenu(u"LocationIQ GeoCoding", self.settingsAction)
```
Comment (gopi-ar) [inline bulkNominatim.py:73]: there are 2 setting menus added to the UI, one's a dummy

**Review by gopi-ar — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (theju) [reply readme.md:15]: The image was updated earlier. Did I miss something?

File: bulkNominatim.py (hunk 4 above)
Comment (theju) [reply bulkNominatim.py:73]: I am sorry I don't get what you mean by one's a dummy?

**Pull request merged**